activemodel/lib/active_model/errors.rb

```diff
@@ -382,10 +382,21 @@ def add_on_blank(attributes, options = {})
     end
 
     # Returns +true+ if an error on the attribute with the given message is
-    # present, +false+ otherwise. +message+ is treated the same as for +add+.
+    # present, or +false+ otherwise. +message+ is treated the same as for +add+.
     #
     #   person.errors.add :name, :blank
     #   person.errors.added? :name, :blank # => true
+    #   person.errors.added? :name, "can't be blank" # => true
+    #
+    # If the error message is a symbol and requires an option, then it returns
```

Comment:
Hi Alex, we still need to drop the "error message is symbol" part, it applies to both string and symbol versions.